### ISSUE TITLE
Google firestore transactional decorator shows up as non-existent

### ISSUE THREAD

**Issue by Duncanr-glitch** [pylint-dev/pylint#8853]
### Bug description

When I use the firebase_admin's firstore transactional decorator, it works perfectly well (since I actually have everything installed) but I get a critical error message from pylint.

I haven't seen this error with any other object from any other part of the firebase libraries.

### Command used

```shell
I was using the VSCode extension(https://github.com/microsoft/vscode-pylint)
```

### Pylint output

```shell
Module 'firebase_admin.firestore' has no 'transactional' member
```

### Expected behavior

No error should appear since the member does exist, is installed, and does work when run.

### Pylint version

```shell
Not sure since I was running the extension by itself.
```

**Comment by mbyrnepr2** [comment]
Thanks for the report @Duncanr-glitch. Can you please provide a minimal working code sample that reproduces the issue?
I'm not seeing the `transactional` member in [firebase_admin.firestore](https://github.com/firebase/firebase-admin-python/blob/master/firebase_admin/firestore.py).
